api/specs.proto

```diff
@@ -266,6 +267,14 @@ message ContainerSpec {
 	HealthConfig healthcheck = 16;
 }
 
+// PluginSpec specifies runtime parameters for a plugin.
+message PluginSpec {
```

Comment:
Oh gosh no - I'd like to get something moving.

I don't think this PR has anything controversial and it's replacing a 6 months old PR so I'd rather take baby steps.